centralbank.py

```diff
@@ -13,6 +14,26 @@ def __init__(self):
         self.twelvemonth_CPI = 0
         self.feedback_counter = 0
         self.prices_list = []
+        self.economy_money = money_supply
```

Comment:
I think, the economy's money should not sit at the central bank. The `money_supply` variable in `InsuranceSimulation` is supposed to track the money of the rest of the economy, i.e. all the agents that are not modelled. The central bank is different.

  * We may think about giving the central bank its own budget. But is this currently necessary?
  * Currently, the central bank only needs money to pay aid. This is a cool idea, and it is fine that the central bank in its capacity as regulator decides on this. But the aid should technically be paid by the government (i.e. not the central bank). That is, the central bank could create an obligation for the `InsuranceSimulation` object (representing the government) for each aid payment it wants to effect.
  * We may allow the central bank to change the money supply of the economy. But this is not currently necessary.